### ISSUE TITLE
Explanation of Ticket #68

### ISSUE THREAD

**Issue by JudgmentDay** [chriscoyier/CSS-Tricks-v10#123]
Sorry it took me so long, but I'm making a re-post of Ticket #68 with explanation and screenshots:

Original Issue: Firefox 15.0.1 Footer Area

For Issue 1:
The dark grey background, which was there during relaunch, seems to be missing now. This only occurs when viewing the V10 Blog Post:
http://css-tricks.com/design-v10/.
<a href='http://postimage.org/image/nx9t6bhan/' target='_blank'><img src='http://s11.postimage.org/nx9t6bhan/091012_01.jpg' border='0' alt="091012 01" /></a>

For Issue 2:
You can move your mouse in an area that has no text, and have the link still trigger a hover state:
<a href='http://postimage.org/image/j3v3ykr4f/' target='_blank'><img src='http://s11.postimage.org/j3v3ykr4f/091012_02.jpg' border='0' alt="091012 02" /></a>

Basically what the second screenshot conveys is that the mouse can be hovered outside of the text link's hit state area, but the link will still be triggered. The hit state seems to happen in a 1-2px based span tag. Not sure if this is the intent of the functionality, but it doesn't seem right.

These issues are happening on a Windows Vista Business SP2 for whatever it's worth.

**Comment by chriscoyier** [comment]
totally got you, thanks. will fix soon.

**Comment by chriscoyier** [comment]
Background thing I can't seem to replicate. Seems just like an image isn't loading, which is likely just a CDN failure or something randomish. 2 == fixed.
